src/components/modules/auth/ui/forms/ForgotPassword.tsx

```
@@ -59,8 +59,9 @@ export function ForgotPasswordForm({
                       </svg>
                     </div>
                     <p className="text-sm text-muted-foreground">
-                      If an account exists with that email, we've sent a password reset link.
-                      Please check your inbox and spam folder.
+                      If an account exists with that email, we've sent a
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)